packages/next/build.mjs

```diff
@@ -0,0 +1,17 @@
+import { join } from 'node:path';
+import { readFileSync } from 'node:fs';
+import { esbuild } from '../../utils/build.mjs';
+import buildEdgeFunctionTemplate from './scripts/build-edge-function-template.js';
```

Comment:
Does this need to be `mjs`?

Reply:
Could be, but doesn't need to be. You can import commonjs files just fine.